compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1261,13 +1261,20 @@ rustc_queries! {
         desc { |tcx| "looking up function parameter names for `{}`", tcx.def_path_str(def_id) }
         separate_provide_extern
     }
+
     /// Gets the rendered value of the specified constant or associated constant.
     /// Used by rustdoc.
     query rendered_const(def_id: DefId) -> &'tcx String {
         arena_cache
         desc { |tcx| "rendering constant initializer of `{}`", tcx.def_path_str(def_id) }
         separate_provide_extern
     }
+    /// Gets the rendered precise capturing args for an opaque for use in rustdoc.
+    query rendered_precise_capturing_args(def_id: DefId) -> Option<&'tcx [Symbol]> {
+        desc { |tcx| "computing precise capturing args for `{}`", tcx.def_path_str(def_id) }
```

Comment:
```suggestion
        desc { |tcx| "rendering precise capturing args for `{}`", tcx.def_path_str(def_id) }
```

maybe? similar to `rendered_const`'s descr